.github/workflows/wheels-build.yml

```diff
@@ -73,7 +73,7 @@ jobs:
         os:
           - ubuntu-latest
           # Used for the x86_64 builds.
-          - macos-13
+          - macos-15-intel
```

Comment:
Does this runner exist? I couldn't see its docs at https://github.com/actions/runner-images/